### PR TITLE
Add some docs for running kind on GCB

### PR DESCRIPTION
**Pull request by hoegaarden**
@BenTheElder

### CODE REVIEW

**Comment by BenTheElder** [comment]
sorry I missed this! thanks!!

File: gcb.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  - src/kind
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: bash
+  dir: bin
+  args:
+  - -c
+  - |
+    curl -L https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl > kubectl
+    chmod 0750 kubectl
+- name: golang
```
Comment (BenTheElder) [inline gcb.md]: we should consider using the makefile to build (and get version info) or downloading a release binary

however, given the delay on this, I'd say we can think about that in a follow-up

Comment (hoegaarden) [reply gcb.md]: Which Makefile are you thinking about? sigs.k8s.io/kind/Makefile?

I can look into that.

**Comment by hoegaarden** [comment]
Generally, I think we (or at least I) want to run some tests with the help of kind on GCB as part of the release process (e.g. running a cluster after we ran the branchff). I would guess that we get a better idea of how to successfully/easily run kind on GCB by doing that. I guess we/I will report back here.

**Pull request merged**